### PR TITLE
add 228. Summary Ranges

### PR DESCRIPTION
**Pull request by kobakobashu**
time : 25m

### CODE REVIEW

**Comment by kobakobashu** [comment]
Please check my code.

File: 228_summary-ranges.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        pre_num = start_num = nums[0]
+        summary_ranges = []
+        for i in range(1, len(nums)):
+            if pre_num + 1 != nums[i]:
+                if start_num == pre_num:
+                    summary_ranges.append(str(pre_num))
+                else:
+                    summary_ranges.append(str(start_num)+"->"+str(pre_num))
+                start_num = nums[i]
+            pre_num = nums[i]
```
Comment (ryuji0123) [inline 228_summary-ranges.py:26]: insert one line

**Review by ryuji0123 — CHANGES_REQUESTED** [review]

**Comment by kobakobashu** [comment]
I inserted one line.  Please check my code.

**Review by tkkawa — APPROVED** [review]
LGTM

**Review by ryuji0123 — APPROVED** [review]

**Pull request merged**